### ISSUE TITLE
1.19 Update

### ISSUE THREAD

**Issue by JRCLARK25** [QuiltServerTools/Interdimensional#23]
Curious as 1.19 approaches if this mod is still being updated. Use it on our server and would hate to have to lose this awesome feature!

**Comment by Henry-Hiles** [comment]
This would be amazing.

**Comment by TheSabotender** [comment]
agreed, I downloaded the code and have been trying to update it myself, but i've never used Kotlin before and i cant get it to compile :/

**Comment by Henry-Hiles** [comment]
Yes, it may need to be updated, not sure.

**Comment by Henry-Hiles** [comment]
Or it may just be a version bump but i doubt that

**Comment by YoSoyMrQ** [comment]
would be great to have a 1.19.2 version

**Comment by snatella** [comment]
It's more fundamental unfortunately. How biomes, dimensions etc were handled looks to have changed. StructureConfig is gone entirely.

I've spent quite a lot of hours which I've committed to my fork as v1.1.2-1.19.2 but it's still borked. What works: GUI (mostly) and commands, and portal linking. What doesn't work: **Save loading crashes on startup** (take backups if you want to fiddle), and dimensions don't generate at all despite "creating".

I'm at the limit of my talent though as a many years out of date JavaEE engineer.

Beware if you take a stab at it from master, it was left by the authors in a WIP rewrite state, I discovered this after spending a couple of hours on it, started afresh from v1.1.1 which was less for my overwhelmed self to unravel.

**Comment by DecDuck** [comment]
@snatella Have you had any luck since then? I'm looking for a Multiverse alternative, and this seems perfect except it's stuck in 1.17

**Comment by SXDSL** [comment]
> @snatella Have you had any luck since then? I'm looking for a Multiverse alternative, and this seems perfect except it's stuck in 1.17

Agree. I would also like to see this updated. Options don't look good for Fabric 1.19.
